src/odemis/acq/stream/test/stream_sparc2_test.py

```
@@ -0,0 +1,191 @@
+
+        # dwell time of sems shouldn't matter
+        mcs.emtDwellTime.value = 1e-6  # s
+        mcs.repetition.value = (500, 700)
+        pca.period.value = 4900e-6  # ~every 10 lines
+        exp_pos, exp_pxs, exp_res = roi_to_phys(mcs)
+
+        # Start acquisition
+        timeout = 1 + 1.5 * sms.estimateAcquisitionTime() + (0.3 * 700 / 10)
+        logging.debug("Expecting acquisition of %g s", timeout)
+        start = time.time()
+        for l in sms.leeches:
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)